src/schema.rs

```diff
@@ -400,9 +407,11 @@ pub struct ItemReceivedOfferData {
     pub created_date: DateTime<Utc>,
     /// Timestamp of when the offer will expire.
     pub expiration_date: DateTime<Utc>,
-    /// Creator of the offer.
+    /// Creator of the listing.
```

Comment:
Is the `maker` of `item_received_offer` the one who created the offer, or the original listing's creator? The docs are totally unclear about this - have you tested this yourself?